dvc/command/experiments/gc.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import logging
+
+from dvc.command.repro import CmdRepro, append_doc_link
+from dvc.exceptions import InvalidArgumentError
+from dvc.ui import ui
+
+logger = logging.getLogger(__name__)
+
+
+def _raise_error_if_all_disabled(**kwargs):
+    if not any(kwargs.values()):
+        raise InvalidArgumentError(
+            "Either of `-w|--workspace`, `-a|--all-branches`, `-T|--all-tags` "
+            "or `--all-commits` needs to be set."
+        )
+
+
+class CmdExperimentsGC(CmdRepro):
```

Comment:
```suggestion
class CmdExperimentsGC(CmdBase):
```

Looks like this was a pre-existing bug? It didn't affect anything, but `dvc exp gc` should not need to inherit anything specific from `dvc repro`